features/fixtures/maze_runner/Assets/Scripts/Main.cs

```diff
@@ -34,23 +34,37 @@ public class Main : MonoBehaviour
     [DllImport("NativeCrashy")]
     private static extern void crashy_signal_runner(float num);
 
+#endif
```

Comment:
Looks like the additions to this file are causing failures in the macOS/Windows tests due to differing line numbers.

Reply:
Fixed